woocommerce/woocommerce-gutenberg-products-block
# Add dynamic rendering + SSR to Products By Category block

See #586 – this updates the Products by Category block to use the new dynamic & ServerSideRender approach to display products.

### Screenshots

The CSS is still a little off for these blocks, but they render correctly:

![Screen Shot 2019-05-22 at 4 49 25 PM](https://user-images.githubusercontent.com/541093/58207972-c94f8c00-7cb1-11e9-84fd-ffe3e9a05742.png)

### How to test the changes in this Pull Request:

1. Add a Products by Category block to a post before activating this PR
2. Build the PR
3. View your existing post, the block should still render correctly
4. Add a new Products by Category block to a post, it should work correctly – show the right products, allow for selecting new categories in the sidebar, toggling the content, etc.
5. View your post on the frontend, it should display same as the editor preview
